src/Pages/Signup/Signup.js

```diff
@@ -0,0 +1,156 @@
+import React from "react";
+import { withRouter } from "react-router-dom";
+import "./Signup.scss";
+
+class Signup extends React.Component {
+  constructor() {
+    super();
+    this.state = {
+      email: "",
+      pw: "",
+      rePw: "",
+      nickName: "",
+      emailAlert: true,
+      pwAlert: true,
+    };
+  }
+
+  handleInputValue = (e) => {
+    const { id, value } = e.target;
+    this.setState({
+      [id]: value,
+    });
+  };
+
+  isEmailValidate = (e) => {
+    e.preventDefault();
+    const { email } = this.state;
+    const checkEmail = email.indexOf("@") !== -1 && email.indexOf(".") !== -1;
```

Comment:
```suggestion
    const checkEmail = email.includes("@") && email.includes(".");
```
includes메서드 활용으로 좀 더 명확한 표현이 가능할 것 같습니다 해인님!

Reply:
네 감사합니다!